contributions/catalog/webapps/static/css/style.css

```diff
@@ -78,24 +79,94 @@ html * {
     clip-path: polygon(0 50%, 100% 0, 100% 100%, 0% 100%);
 }
 
-input[type=text], select, textarea {
-    color: #090000;
+label {
+    font-size: 16px;
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    letter-spacing: normal;
+    height: 20px;
+    color: #13294b;
+}
+
+input[type=text], input[type=url], select, textarea {
+    color: #404040;
     width: 100%;
-    padding: 12px;
-    border: 1px solid #ccc;
+    padding: 5px;
+    border: 1px solid #002855;
+    resize: vertical;
+    background-color: #fff;
     border-radius: 4px;
+}
+
+input[type=number], input[type=date]{
```

Comment:
Is there a difference in the styles of different inputs like numbers, dates, etc.? Do we need separate style classes for that?

Reply:
input type text , url and textarea have width 100%. input type numbers and dates have other specified widths.

Reply:
Thanks for clarifying, Minu.